run.sh

```diff
@@ -22,6 +22,7 @@ WAMR=$BASE/deps/wasm-micro-runtime
 EMSDK=$BASE/deps/emsdk
 
 setup_deps() {
+  clean_all_wasm
```

Comment:
I'd prefer not to clean on every build; the wasm builds were generally the slowest (which is why I have the "newer file" detection below).